Add note to laravel-activitylog integration (#177)
stancl/tenancy-docs: source/docs/v3/integrations/spatie.blade.md

## **laravel-activitylog** {#laravel-activitylog}

> Note: Activity's subject_id has to be an integer. E.g., to use the LogsActivity trait on a model, the model's ID has to be an integer.

Author: ```suggestion
> Note: Activity's subject_id has to be an integer. E.g., to use the LogsActivity trait on a model, the model's ID has to be an integer.
```

> > Note: Activity's subject_id has to be an integer. E.g., to use the LogsActivity trait on a model, the model's ID has to be an integer.

Reviewer: ```suggestion
> Note: Activity's subject_id has to be an integer. For example, to use the LogsActivity trait on a model, the model's ID has to be an integer.
```

Also I think the comment should be more concrete? It just says "on a model", and "the model", but it doesn't say that this is the tenant model?

Author: It doesn't necessarily have to be a Tenant model, "the model" refers to the model in question (the model we want to use the trait on).

But yeah, I guess we should be concerned about the Tenant model only because that's what Tenancy provides

Reviewer: The note should say that:
- *the model* needs to have an integer ID
- therefore, *the Tenant model* needs to be modified to have an integer ID in our case

And ideally, mention that extra security measures are recommended with integer IDs for tenants, because the IDs become enumerable. uuids are safe against enumeration attacks.

Author: Updated the note
https://github.com/stancl/tenancy-docs/compare/526f4eb1b8b42d177402e3c4058a45eb8692dd4e..Add-note-to-laravel-activitylog-integration

Reviewer: If this is resolved, you can close the review.

## **laravel-activitylog** {#laravel-activitylog}

> > Note: The ID of the model who's the subject of Activity needs to be an integer. We recommend extra security measures when using integer IDs for tenants. Because the IDs become enumerable, they get vulnerable to enumeration attacks (which UUIDs are safe against).

Reviewer: > who

models are inanimate

And the first sentence seems pretty clunky and complex

Author: ```suggestion
> Note: The model needs to have an integer ID to have activity. We recommend extra security measures when using integer IDs for tenants. Because the IDs become enumerable, they get vulnerable to enumeration attacks (which UUIDs are safe against).
```

Is this better?

Reviewer: > to have activity

is weird

Reviewer: I mean, you don't have to write about performing activity or having activity. You should just explain the package's requirements. This isn't about needing an integer ID to perform activity. It's about the activity package requiring integer IDs. So just explain that.

Reviewer: I'd go with the latter suggestion. If you want to make it more accurate, you could make it say something like "The package requires **logged** models to have integer IDs" (without the emphasis).

Author: "logged models" seems the best
